docs/src/examples/components/Dropdown/Types/DropdownExampleSingleSelection.shorthand.steps.js

```diff
@@ -0,0 +1,7 @@
+module.exports = [
+  steps => steps.click('.ui-dropdown .ui-button').snapshot('Shows list'),
```

Comment:
please, do not use hardcoded class names - use string interpolation and `className` prop of corresponding component instead. This will ensure tests' robustness

Reply:
will take care of this when we refactor for TS